core/src/services/gcs/core.rs

```diff
@@ -236,13 +238,23 @@ impl GcsCore {
         if let Some(storage_class) = &self.default_storage_class {
             let mut multipart = Multipart::new();
 
+            let mut meta_data = HashMap::new();
```

Comment:
We can built this hashmap outside. And only use multipart when this hashmap is not empty.

Reply:
ok